content/projects/nodejs/ajax/_index.md

```diff
@@ -45,10 +50,14 @@ Also, make sure that if you create any new visitors then they are visible on the
 
 ## Add a visitor
 
-Add a button labelled "New Visitor". This button should redirect the user to the new visitor form that was created in an earlier step in this project. 
+Add a button labelled "New Visitor". This button should redirect the user to the new visitor form that was created in an earlier step in this project.
+
+Update the "thank you page" so that it only displays for 3 seconds and then redirects the user back to the visitors table. Of course the new visitor should be visible in the table.
+
+## Up for a challenge?
 
-Update the "thank you page" so that it only displays for 3 seconds and then redirects the user back to the visitors table. Of course the new visiter should be visible in the table.
+So far, you've utilized only three API end points(`POST`, `GET` and `DELETE`). Can you figure out how to make use the `PUT` endpoint? The implementation here is completely up to you.
```

Comment:
This language is a bit odd. Those aren't endpoints. Those are verbs.

How about just saying something like:

"""
We can now add, list and delete visitor objects.  Add a feature that lets the user update a Visitor's details. Pay close attention to usability - how can you make this feature easy and obvious to the user?
"""